### PR TITLE
Feature/lightbar

### PR DESCRIPTION
**Pull request by MishkaMN**
# PR Details
## Description

Fixed docker dir issue where CMake could not find cav_driver_utils when invoked from docker. 
Had to put the package in a separate folder from docker folder. 

## How Has This Been Tested?

Tested on Fusion and both Lexus

## Types of changes

- [X] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](Contributing.md) 
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+FROM usdotfhwastol/carma-base:3.6.0 as setup
+
+RUN mkdir ~/src
+COPY --chown=carma . /home/carma/src/
+RUN ~/src/docker/checkout.sh
+RUN ~/src/docker/install.sh
+FROM usdotfhwastol/carma-base:3.6.0
```
Comment (msmcconnell) [inline Dockerfile]: carma-base is not on version 3.7.0

File: lightbar_driver/CMakeLists.txt [hunk 2]
```diff
@@ -3,8 +3,8 @@ project(lightbar_driver)
 
 ## Compile as C++11, supported in ROS Kinetic and newer
 add_compile_options(-std=c++11)
-set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
-set( CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall")
+#set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
+#set( CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall")
```
Comment (msmcconnell) [inline lightbar_driver/CMakeLists.txt]: These should not be commented out as they enable warning messages at build time

File: lightbar_driver/src/.vscode/c_cpp_properties.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "configurations": [
```
Comment (msmcconnell) [inline lightbar_driver/src/.vscode/c_cpp_properties.json]: looks like you accidentally committed some of your vscode files.

**Comment by msmcconnell** [comment]
Make sure every PR you put up is linked to an issue on the CARMAPlatform repo. You can make a new one if needed

**Comment by MishkaMN** [comment]
Uploaded the PR changes.

**Pull request merged**